Extension/SourceEditorCommand.swift

```diff
@@ -4,24 +4,21 @@ import XcodeKit
 class SourceEditorCommand: NSObject, XCSourceEditorCommand {
 
     func perform(with invocation: XCSourceEditorCommandInvocation,
-                 completionHandler: @escaping (Error?) -> Swift.Void) {
+        completionHandler: @escaping (Error?) -> Swift.Void) {
 
         let uti = invocation.buffer.contentUTI
-        if uti != "com.apple.dt.playground" && uti != "public.swift-source" && uti != "com.apple.dt.playgroundpage"{
+        guard uti == "com.apple.dt.playground" || uti == "public.swift-source" || uti == "com.apple.dt.playgroundpage" else {
```

Comment:
Hmm, what bug are you talking about? In any case the UTI checking code needs to be rewritten; I'll do this after this+a couple other pull requests are merged.